### PR TITLE
Ref gitlab conn id issue 5

### PR DESCRIPTION
**Pull request by warren830**
# Summary

add conn id  and e2e to issues

### Does this close any open issues?
Please mention the issues here.

### Screenshots
image
### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: plugins/gitlab/tasks/issue_convertor.go [hunk 1]
```diff
@@ -94,7 +88,7 @@ func ConvertIssues(taskCtx core.SubTaskContext) error {
 				domainIssue.Status = ticket.DONE
 			}
 			boardIssue := &ticket.BoardIssue{
-				BoardId: boardIdGen.Generate(projectId),
+				BoardId: boardIdGen.Generate(data.Options.ConnectionId, projectId),
```
Comment (likyh) [inline plugins/gitlab/tasks/issue_convertor.go:91]: maybe can replace `projectId` with `data.Options.ProjectId` to keep code consisont like above (Line 49).

File: plugins/gitlab/tasks/issue_extractor.go [hunk 2]
```diff
@@ -134,7 +134,8 @@ func ExtractApiIssues(taskCtx core.SubTaskContext) error {
 				set of data to be process, for example, we process JiraIssues by Board
 			*/
 			Params: GitlabApiParams{
-				ProjectId: data.Options.ProjectId,
+				ProjectId:    data.Options.ProjectId,
+				ConnectionId: data.Options.ConnectionId,
```
Comment (likyh) [inline plugins/gitlab/tasks/issue_extractor.go:138]: use CreateRawDataSubTaskArgs?

**Review by likyh — CHANGES_REQUESTED** [review]

File: plugins/gitlab/tasks/issue_convertor.go (hunk 1 above)
Comment (warren830) [reply plugins/gitlab/tasks/issue_convertor.go:91]: fix

File: plugins/gitlab/tasks/issue_extractor.go (hunk 2 above)
Comment (warren830) [reply plugins/gitlab/tasks/issue_extractor.go]: fix

**Review by likyh — APPROVED** [review]
LGTM

**Review by abeizn — APPROVED** [review]

**Pull request merged**